lib/node_modules/@stdlib/complex/float64/base/div/examples/index.js

```diff
@@ -18,23 +18,14 @@
 
 'use strict';
 
-var Complex128 = require( '@stdlib/complex/float64/ctor' );
-var discreteUniform = require( '@stdlib/random/base/discrete-uniform' );
+var Complex128Array = require( '@stdlib/array/complex128' );
+var discreteUniform = require( '@stdlib/random/array/discrete-uniform' );
+var logEachMap = require( '@stdlib/console/log-each-map' );
 var cdiv = require( './../lib' );
 
-function randomComplex() {
-	var re = discreteUniform( -50, 50 );
-	var im = discreteUniform( -50, 50 );
-	return new Complex128( re, im );
-}
+// Generate arrays of random values:
+var z1 = new Complex128Array( discreteUniform( 200, -50, 50 ) );
+var z2 = new Complex128Array( discreteUniform( 200, -50, 50 ) );
 
-var z1;
-var z2;
-var z3;
-var i;
-for ( i = 0; i < 100; i++ ) {
-	z1 = randomComplex();
-	z2 = randomComplex();
-	z3 = cdiv( z1, z2 );
-	console.log( '(%s) / (%s) = %s', z1.toString(), z2.toString(), z3.toString() );
-}
+// Perform division for each element of arrays
```

Comment:
@Planeshifter Please make sure the comments are consistent and have the correct ending. I just pushed a change updating the comments for `mul`. These and other comments should follow suit.